guides/blending/client-side-blending/README.md

```diff
@@ -0,0 +1,159 @@
+# Client-Side Blending
+
+When merging responses from different apps with [server-side blending](server-side-blending), the views are stacked on top of each other. In most cases, that's not what we want. Instead, we would like to compose elements in the views to make the result look like one page. That's what client-side blending does:
+
+![Blending comparison](/assets/SimpleBlendingDemo.PNG)
```

Comment:
I like both this image and the diagram "BlendingComparison2.png". Can we use both?